CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+## Contributing
+
+| Component | Build Status |
+| --------- | ------------ |
+| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
+| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
+
+### Prerequisites:
+  Install VS 2017 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+
+### To build:
+=======
+#### If you have recently cloned the repo or did a pull from upstream or did git clean -xdf you need to the following before you do anything else in order to build your projects
+
+ 1. Open **elevated** VS 2017 command prompt
+ 2. Navigate to repository root directory
+ 3. Skip verification for the following dlls
+    - Sn -Vr tools\bootstrapTools\taskBinaries\Microsoft.Azure.Build.BootstrapTasks.dll
+    - Sn -Vr tools\SdkBuildTools\tasks\net46\Microsoft.Azure.Sdk.Build.Tasks.dll
+ 4. Execute MsBuild.exe build.proj (this will pull all the build related tools needed to build the repo)
+ 5. Follow below steps to start building your repo/project
+
+#### If you are building from VS, add a nuget feed source that points to < root >\tools\LocalNugetFeed directory
+ 1. Open any solution, eg "SDKs\Compute\Compute.sln"
+ 2. Build solution from VS
+ 
+#### Full Build from command line
+
+ 1. Open VS 2017 command prompt
+ 2. Navigate to repository root directory
+ 3. Invoke **msbuild** build.proj /t:Build
+ will Build
+ ##### *Build* without any scope will build all SDK's and create nuget packages.
+
+#### Create single nuget package
+In order to build one package and run it's test
+`msbuild build.proj /t:CreateNugetPackage /p:scope=SDKs\Compute`
+Nuget package will be created in root directory under \binaries\packages
+ 
+### To run the tests:
+Using Visual Studio:
+  - Build.
+  - "Test Explorer" window will get populated with tests. Select test and Run/Debug.
+
+Using the command line:
+msbuild .\build.proj /t:"Runtests" /p:Scope=SDKs\Compute
+in the above example RunTests will build and run tests for Compute only
+or
+dotnet test SDKs\Compute\Compute.Tests\Compute.Tests.csproj
+
+  - Refer to the "To build" section to get the command window set up.
+  - Invoke "RunTests" target from "Build.proj". RunTests will build and run tests 
+        *msbuild build.proj /t:RunTests /p:scope=SDKs\Compute*
+
+## To on-board new libraries
```

Comment:
Should we put information about the new repo structure here?

Reply:
This is not part of this PR, i.e. this file is text just moved from the old readme. But please file an issue if you'd like something changed in the CONTRIBUTING.MD file.